conferences/ticketing_processing.go

```diff
@@ -0,0 +1,145 @@
+package conferences
+
+import (
+	"context"
+	"fmt"
+
+	"encore.dev/storage/sqldb"
+	uuid "github.com/satori/go.uuid"
+)
+
+// claimSlots claims N slots for an attendee.
+func claimSlots(ctx context.Context, attendee *Attendee, slots ...ConferenceSlot) ([]SlotClaim, error) {
+	tx, err := sqldb.Begin(ctx)
+	if err != nil {
+		return nil, fmt.Errorf("beginning atomic operation: %w", err)
+	}
+	var claims = make([]SlotClaim, 0, len(slots))
+	for i := range slots {
+		slot := slots[i]
+		sc := &SlotClaim{
+			ConferenceSlot: &slot,
+			TicketID:       uuid.NewV4().String(),
+		}
+		sc, err = createSlotClaim(ctx, tx, sc)
+		if err != nil {
+			if atomicErr := sqldb.Rollback(tx); atomicErr != nil {
+				err = fmt.Errorf("%w (also cancelling atomic operation: %v)", err, atomicErr)
+			}
+			return nil, fmt.Errorf("Claiming a slot: %w", err)
```

Comment:
Errors strings typically do not start with a capital letter (https://github.com/golang/go/wiki/CodeReviewComments#error-strings).

```suggestion
			return nil, fmt.Errorf("claiming a slot: %w", err)
```

This is a general comment.